.github/workflows/link-check.yml

```diff
@@ -13,11 +13,11 @@ jobs:
 
     - name: Run the link checker
       id: check
-      uses: "iterative/link-check-git-diff.action@master"
+      uses: "iterative/link-check.action@master"
```

Comment:
love the new name!

Reply:
Thanks! With the new filesystem feature, I'm more comfortable naming it something more generic.